### PR TITLE
fix(added-license-context-color): added license context color, some b…

### PR DESCRIPTION
**Pull request by jyasveer**
…ug fixes

### CODE REVIEW

**Comment by invincibleJai** [comment]
@arunkumars08  have a look once

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 @Component({
     selector: 'f8-app',
     templateUrl: './app.component.html'
 })
 
 export class AppComponent implements OnInit {
-    public stackUrl: string = 'https://recommender.api.openshift.io/api/v1/stack-analyses/9f9a8e4c026c4f6590ed42149c97995d';
+    public stackUrl: string =
+    'https://recommender.api.openshift.io/api/v1/stack-analyses/a0cceacbcac54d4b8d6897ce00b726eb';
```
Comment (joshuawilson) [inline src/app/app.component.ts]: Why did you move this to a new line? It is harder to read, at least indent it.

Also, this appears to be 4 space indents. We are using 2 spaces per indent.

Comment (jyasveer) [reply src/app/app.component.ts]: i moved this to new line to fix the 100 characters per line lint error. Some files have 4 space indent and some 2 space indent. I have kept the same indent as it was for this file.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import { Component, OnInit } from '@angular/core';
```
Comment (arunkumars08) [inline src/app/app.component.ts]: Could you not change the existing indentation for this file? It goes as a file change in git.

Comment (jyasveer) [reply src/app/app.component.ts]: let us do this right. we can keep the code readable. its something to fix the indentation.
